--- syslinux/syslinux.cfg	2018-05-03 17:18:30.000000000 +0100
+++ /home/aparicio/repos/labs-bootstrap/syslinux.cfg	2018-05-03 17:08:43.849046873 +0100
@@ -9,6 +9,6 @@
 PROMPT 0
-TIMEOUT 900
-ONTIMEOUT rescuecd_std
+TIMEOUT 100
+ONTIMEOUT bootstrap
 
-MENU DEFAULT rescuecd_std
+MENU DEFAULT bootstrap
 MENU TABMSG Press [TAB] to edit options or <F2>,<F3>,<F4>,<F5>,<F6>,<F7> for help
@@ -31,2 +31,10 @@
 
+LABEL bootstrap
+MENU LABEL 0) Labs bootstrap initramfs
+KERNEL labs-bootstrap-kernel
+INITRD labs-bootstrap-initramfs
+TEXT HELP
+Initramfs to bootstrap our lab workstations by torrent.
+ENDTEXT
+
 LABEL rescuecd_std
@@ -34,3 +42,3 @@
 KERNEL ifcpu64.c32
-APPEND rescue64 scandelay=5 -- rescue32 scandelay=5
+APPEND rescue64 scandelay=5 setkmap=pt -- rescue32 scandelay=5 setkmap=pt
 TEXT HELP
